### ISSUE TITLE
Unable to access certain PDFs in new UI

### ISSUE THREAD

**Issue by JulieJager** [syncids/syncids-ui#271]
Production & Development & Old UI.
finchmaloney.com
I am not able to view the foreign PDFs shown below.  The PDF icons resolved to red in the old UI too but nothing is there.  This is unusual.  Also, there is description in the NPL section that does not belong.  Note: the PDF icon is not shown with it.  Another example immediate below. View set to Uncited, but UI is showing Test4 which is cited. This could be related to attempting to enter a duplicate.  PDF icon is missing as well.
![capture](https://cloud.githubusercontent.com/assets/17907226/16015010/66c6499e-3159-11e6-8c89-66f944c33fe7.JPG)

![capture](https://cloud.githubusercontent.com/assets/17907226/16014018/1cdec0a8-3155-11e6-87cf-916ae270c536.JPG)

**Comment by ntgn81** [comment]
@yneves , the data on the new NPL doc seems like things are not getting cleared properly. Like when the value is undefined, it's not clearing out.

**Comment by ntgn81** [comment]
@JulieJager , as for the pdf not visible. The same behaviour is in old UI, the PDF files are gone, or not accessible for some reason. Just the new UI feeds it directly through to google pdf viewer.

The old UI would show this:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>NoSuchKey</Code>
  <Message>The specified key does not exist.</Message>
  <Key>finchmaloneycom/701/02003247154.pdf</Key>
  <RequestId>10EC9D45D60D8CBF</RequestId>
  <HostId>cHYeJsgrf97PQ8tSqmuyJDYSWCnxuUFLKuWudKE/Ang/vR7QvKRhRIGy+kOTfoDcVvjTcUqLK6I=</HostId>
</Error>
```

**Comment by yneves** [comment]
@ntgn81 looks like the issue with `key` i fixed here #277
